integration_testing/features/admin/admin-bulk-import-users-classrooms.feature

```diff
@@ -5,32 +5,35 @@ Feature: Admin bulk export of users and classrooms
     Given I am signed in to Kolibri as a facility admin user
       And I am on *Facility > Data* page
 
-  Scenario: See CSV file format needed to import users
-    When I click on "Import" button under *Import and export users* heading
+  Scenario: See CSV the file format requirements
```

Comment:
I don't understand the meaning of this `the`

Reply:
My bad, it should have said `See the CSV file format requirements` :woman_facepalming: .